cmake/macos/compilerconfig.cmake

```diff
@@ -21,8 +21,8 @@ endif()
 set_property(CACHE CMAKE_OSX_ARCHITECTURES PROPERTY STRINGS arm64 x86_64)
 
 # Ensure recent enough Xcode and platform SDK
-set(_obs_macos_minimum_sdk 14.2) # Keep in sync with Xcode
-set(_obs_macos_minimum_xcode 15.1) # Keep in sync with SDK
+set(_obs_macos_minimum_sdk 15.0) # Keep in sync with Xcode
+set(_obs_macos_minimum_xcode 16.0) # Keep in sync with SDK
```

Comment:
Does this need to be bumped to Xcode 16.1 and the associated SDK version, or do we consider 16.0 "recent enough"?

Reply:
We'd only need to bump if we need a feature only available in a more recent platform SDK (and the platform SDK version is usually coupled with a new Xcode version).